src/mame/namco/namcos10_exio.h

```diff
@@ -29,6 +33,9 @@ class namcos10_exio_base_device : public device_t
 	virtual void device_start() override;
 
 	const uint8_t m_ident_code;
+
+	uint16_t m_bus_req;
+	uint16_t m_ctrl;
```

Comment:
Why have these been moved here?  from a glance at the diff, it looks like they’re still part of `namcos10_exio_device` and not used by this class.

Reply:
I haven't found any usage of it that'd matter for namcos10_exio_base_device so it's technically not needed in the base class, but my reasoning for it is that I believe the CPLD is responsible for these registers and the CPLD is on both the basic EXIO board (without the additional CPU or RAM) and the fully populated EXIO board (with the additional CPU and RAM). So I figured it would make more sense to move those to the base device and then allow the fully featured EXIO and MGEXIO override those instead of having the functions in both classes.

I can remove it from the base class if you think that would be cleaner. It's not necessary for it to be in the base class.

Reply:
It doesn’t really make sense to have it in the base class when it isn’t being registered for save states there, or used there in any way at all.

Reply:
Ok, I'll revert that part of the change then.